core/src/file/qt_fs_browser.cc

```diff
@@ -40,21 +41,35 @@ qt_fs_browser::~qt_fs_browser() {}
  *
  *  @return List of elements within the directory.
  */
+#include <iostream>
 fs_browser::entry_list qt_fs_browser::read_directory(
```

Comment:
We should change this name now :-) : qt_fs_browser...

Reply:
done stl_fs_browser !